cmd/notary/keys.go

```diff
@@ -127,11 +128,7 @@ func (k *keyCommander) GetCommand() *cobra.Command {
 	cmdRotateKey.Flags().BoolVarP(&k.rotateKeyServerManaged, "server-managed", "r",
 		false, "Signing and key management will be handled by the remote server. "+
 			"(no key will be generated or stored locally) "+
-			"Can only be used in conjunction with --key-type.")
-	cmdRotateKey.Flags().StringVarP(&k.rotateKeyRole, "key-type", "t", "",
-		`Key type to rotate.  Supported values: "targets", "snapshot". `+
-			`If not provided, both targets and snapshot keys will be rotated, `+
-			`and the new keys will be locally generated and stored.`)
+			"Can only be used for timestamp and snapshot roles, and MUST be used for the timestamp role.")
```

Comment:
not sure if we should use the all-caps here.  Maybe we can reword, what do you think of this?

`Required for timestamp role, optional for snapshot role`